ext/search_kit/tests/phpunit/api/v4/SearchDisplay/EntityDisplayTest.php

```diff
@@ -79,11 +89,19 @@ public function testEntityDisplay() {
       ->execute()->first();
 
     $schema = \CRM_Core_DAO::executeQuery('DESCRIBE civicrm_sk_my_new_entity')->fetchAll();
-    $this->assertCount(5, $schema);
+    $this->assertCount(7, $schema);
     $this->assertEquals('_row', $schema[0]['Field']);
     $this->assertStringStartsWith('int', $schema[0]['Type']);
     $this->assertEquals('PRI', $schema[0]['Key']);
 
+    // created_date and modified_date should be NULLable and have no default/extra
```

Comment:
I'm not sure I understand this. When would created_date for anything be null?

Reply:
Or are you just using those field names as examples - it might be better to call them something else where it would make sense that they could be null.

Reply:
Agree this is confusing - this is specifically in the context of an `SKEntity` based on `Contact`, which has a NULLable `created_date` (presumably because it was added after the fact).

(Also, thanks for reviewing/merging this!)